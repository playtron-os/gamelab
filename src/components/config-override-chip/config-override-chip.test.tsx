import React from "react";
import { ConfigOverrideChip } from ".";
import { render } from "@testing-library/react";

it("ConfigOverrideChip - renders correctly", () => {
  const { container, asFragment } = render(
    <ConfigOverrideChip label="Test" selected={false} />
  );

  expect(container).not.toBeEmptyDOMElement();
  expect(asFragment()).toMatchSnapshot();
});
