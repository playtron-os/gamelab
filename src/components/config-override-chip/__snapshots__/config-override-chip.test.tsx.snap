// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfigOverrideChip - renders correctly 1`] = `
<DocumentFragment>
  <div
    class="config-override-chip-rct-component group p-3 outline-1 rounded-[--radii-md] outline-[--stroke-subtle] text-base hover:outline flex flex-row gap-2 cursor-pointer"
  >
    <span
      class="flex-grow truncate w-1/2"
    >
      Test
    </span>
  </div>
</DocumentFragment>
`;
